### PR TITLE
[mesheryctl] Creating meshconfig context profiles: default slim

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: leecalcote <leecalcote@gmail.com>

**Notes for Reviewers**

This PR changes the default meshconfig to include only a single Meshery adapter.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: mesheryctl/pkg/utils/helpers.go [hunk 1]
```diff
@@ -188,15 +188,18 @@ var CfgFile string
 
 // TODO: add "meshery-perf" as a component
 
-// ListOfComponents returns the list of components available
-var ListOfComponents = []string{"meshery-app-mesh", "meshery-istio", "meshery-linkerd", "meshery-consul", "meshery-nsm", "meshery-kuma", "meshery-osm", "meshery-traefik-mesh", "meshery-nginx-sm", "meshery-cilium"}
+// allComponentsProfile returns the list of components available
+var allComponentsProfile = []string{"meshery-app-mesh", "meshery-istio", "meshery-linkerd", "meshery-consul", "meshery-nsm", "meshery-kuma", "meshery-osm", "meshery-traefik-mesh", "meshery-nginx-sm", "meshery-cilium"}
```
Comment (Revolyssup) [inline mesheryctl/pkg/utils/helpers.go:192]:
```suggestion
var AllComponentsProfile = []string{"meshery-app-mesh", "meshery-istio", "meshery-linkerd", "meshery-consul", "meshery-nsm", "meshery-kuma", "meshery-osm", "meshery-traefik-mesh", "meshery-nginx-sm", "meshery-cilium"}
```

File: mesheryctl/pkg/utils/helpers.go [hunk 2]
```diff
@@ -188,15 +188,18 @@ var CfgFile string
...
+
+// defaultComponentsProfile returns the list of components available
+var defaultComponentsProfile = []string{"meshery-istio"}
```
Comment (Revolyssup) [inline mesheryctl/pkg/utils/helpers.go:195]:
```suggestion
var DefaultComponentsProfile = []string{"meshery-istio"}
```

File: mesheryctl/pkg/utils/helpers.go [hunk 3]
```diff
@@ -188,15 +188,18 @@ var CfgFile string
...
 
 // TemplateContext is the template context provided when creating a config file
 var TemplateContext = config.Context{
 	Endpoint:   EndpointProtocol + "://localhost:9081",
 	Token:      "Default",
 	Platform:   "kubernetes",
-	Components: ListOfComponents,
+	Components: defaultComponentsProfile,
```
Comment (Revolyssup) [inline mesheryctl/pkg/utils/helpers.go:202]:
```suggestion
	Components: DefaultComponentsProfile,
```

**Review by suhail34 — APPROVED** [review]
Yeah this was what we where talking about today right?

**Pull request closed without merging**